tests/components/rfxtrx/test_init.py

```diff
@@ -198,4 +201,4 @@ def record_event(event):
 
         self.hass.block_till_done()
         assert 1 == len(calls)
-        assert calls[0].data == {"entity_id": "sensor.test"}
+        assert calls[0].data == {"entity_id": "sensor.test_temperature"}
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)